lib/internal/priority_queue.js

```diff
@@ -11,7 +11,8 @@ const {
 // just a single criteria.
 
 module.exports = class PriorityQueue {
-  #compare = (a, b) => a - b;
+  // eslint-disable-next-line no-confusing-arrow
```

Comment:
Can't we avoid this by using parentheses around the entire return value expression or something similar?

Reply:
I wonder why this has to be a private method, it doesn't even make use of `this` anyway...can't this just be a regular function ouside the class? That'll save the cost of private brand-checking, too.

Reply:
Oh right, the method is actually configurable..though I think the default can just still be a function outside the class so that the instantiation doesn't have to include a closure creation.

Reply:
It's possible, but I think that change hardly contributes to performance.

Reply:
we're not golfing :P
```suggestion
  #compare = (a, b) => {
    if (a > b) return 1;
    if (a < b) return -1;
    return 0;
  };
```

Reply:
I thought so if I didn't see golf code here.

https://github.com/nodejs/node/blob/0b5b5edd863c7eea846fdae4218f2b02e04ee31e/lib/internal/readline/promises.js#L94-L97
https://github.com/nodejs/node/blob/0b5b5edd863c7eea846fdae4218f2b02e04ee31e/lib/internal/readline/callbacks.js#L105-L106
https://github.com/nodejs/node/blob/0b5b5edd863c7eea846fdae4218f2b02e04ee31e/lib/internal/http2/core.js#L2612-L2615

This expanding is really necessary?

Reply:
From https://eslint.org/docs/latest/rules/no-confusing-arrow, https://github.com/nodejs/node/pull/44506#discussion_r962193499, and https://github.com/nodejs/node/blob/0b5b5edd863c7eea846fdae4218f2b02e04ee31e/lib/internal/http2/core.js#L2612-L2615, the correct style is to enclose the arrow with parentheses.